hostscripts/soc-ci/soc-ci

```diff
@@ -99,7 +105,8 @@ def _os_mkcloud_where(job_id):
     conf = _config_get()
     server = Jenkins(conf['url'],
                      username=conf['user'],
-                     password=conf['password'])
+                     password=conf['password'],
+                     ssl_verify=conf['ssl_verify'])
```

Comment:
This needs to be added whereever ```Jenkins``` is used. Or add a helper method to get the server object.